app/views/layout.hbs

```diff
@@ -72,11 +72,11 @@
     <!-- Bootstrap core JavaScript
     ================================================== -->
     <!-- Placed at the end of the document so the pages load faster -->
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
+    <script src="http://code.jquery.com/jquery-2.1.4.js"></script>
```

Comment:
It looks like we have the same imports in two files, handlebars,like jade, probably has some support for partials, which would allow you to keep your code more DRY.